README.md

```diff
@@ -1,88 +1,114 @@
-# Svelte Demo
+# Web Component Svelte Demo
 
 ![Travis (.org)](https://img.shields.io/travis/alaskaairlines/AuroSvelteDemo?style=for-the-badge) ![GitHub tag (latest by date)](https://img.shields.io/github/v/tag/AlaskaAirlines/AuroSvelteDemo?style=for-the-badge)
 
-Interested in starting a new project using Auro and Svelte? This demo project is a complete example that includes basic setup. 
+An example Svelte project with Auro Components integrated. This app runs and works in all Alaska-supported browsers. Explore the project source to see the Auro Components being used in a frameworkless environment.
```

Comment:
"frameworkless environment" doesn't apply here. Maybe this was copied from the vanilla JS demo?

Reply:
Yes, yes it was. I'll remove the reference.